### PR TITLE
eip-draft_Pull_Oracle_Interface.md

### PR DESCRIPTION
**Pull request by brendaloya**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your Github username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-draft_Pull_Oracle_Interface.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+eip: xxxx
```
Comment (axic) [inline EIPS/eip-draft_Pull_Oracle_Interface.md]: Please use 2362 and rename the file to eip-2362.md

File: EIPS/eip-draft_Pull_Oracle_Interface.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+title: Pull Oracle Interface
+author: Alliance of Decentralized Oracles (l.loya.b@gmail.com)
+discussions-to: https://github.com/ethereum/EIPs/issues/
```
Comment (axic) [inline EIPS/eip-draft_Pull_Oracle_Interface.md]: This has to be a link to an issue.

File: EIPS/eip-draft_Pull_Oracle_Interface.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+status: DRAFT
```
Comment (axic) [inline EIPS/eip-draft_Pull_Oracle_Interface.md]: This should not be all uppercase, please check EIP-1.

File: EIPS/eip-draft_Pull_Oracle_Interface.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+## Motivation
+
+As the value held by smart contracts that rely on off-chain data to execute grows, the security of oracles systems and the ability of these to be interchangeable and implemented in groups will grow in importance. 
+
+The Ethereum ecosystem currently has many different oracle implementations available, but they do not provide a unified interface. Smart contract systems would be locked into a single set of oracle implementations, or they would require developers to write adapters/ports specific to the oracle system chosen in a given project.
+
+
+## Background
+EIP 1154, for standardizing an oracle interface, was not widely adopted and abandoned as too early and specificaly too broad/general to provide any efficiency gains. This new proposal is a cooperation between current oracle systems and direct users of these. The specification below reflects this collaboration and addresses the main reason why 1154 was not successful.
```
Comment (axic) [inline EIPS/eip-draft_Pull_Oracle_Interface.md]: Please use `EIP-1154` as the name and make it a link to the EIP. Likewise with 1154.

File: EIPS/eip-draft_Pull_Oracle_Interface.md (hunk 1 above)
Comment (brendaloya) [reply EIPS/eip-draft_Pull_Oracle_Interface.md]: Done! thanks for reviewing and the feedback.

File: EIPS/eip-draft_Pull_Oracle_Interface.md (hunk 2 above)
Comment (brendaloya) [reply EIPS/eip-draft_Pull_Oracle_Interface.md]: Done! https://github.com/ethereum/EIPs/issues/2379

File: EIPS/eip-draft_Pull_Oracle_Interface.md (hunk 3 above)
Comment (brendaloya) [reply EIPS/eip-draft_Pull_Oracle_Interface.md]: done

File: EIPS/eip-draft_Pull_Oracle_Interface.md (hunk 4 above)
Comment (brendaloya) [reply EIPS/eip-draft_Pull_Oracle_Interface.md]: updated all instances with EIP-1154 with link

**Comment by brendaloya** [comment]
Hi Alex, Travis ci is running on the old file. I have renamed it and made the necessary updates. I ran eip-validator locally and all seems well (see below). Let me know if there is anyting else I should do.

 eip_validator C:/Users/lloya/Documents/DDA/1Tellor/tellor/projects/eip2/EIPs/EIPS/eip-2362.md



total:1, valid:1, invalid:0, errors:0
        statuses: [["Last Call", 1]]
        types: [["Standards Track", 1]]
        categories: [["ERC", 1]]
        layers: [[nil, 1]]

**Pull request closed without merging**